### ISSUE TITLE
style the footnote reference in non-visual reading test reading-420

### ISSUE THREAD

**Issue by marisademeglio** [daisy/epub-accessibility-tests#89]
https://github.com/daisy/epub-accessibility-tests/blob/revision-non-visual-reading-2023/content/epub30-test-0302/EPUB/xhtml/Non_Visual_Reading_Tests.xhtml#L73

**Comment by marisademeglio** [comment]
I styled the footnote but some changes I made are not reflected in Thorium, I suspect because it overrides certain things. 

**Thorium:**

<img width="596" alt="Thorium reader shows a small footnote reference" src="https://github.com/daisy/epub-accessibility-tests/assets/734730/e22e9679-096b-4c5f-be69-ef3e1077e6df">

**Chrome (standalone):**

<img width="576" alt="Chrome browser shows a big footnote reference" src="https://github.com/daisy/epub-accessibility-tests/assets/734730/19229f2b-ba33-44d2-b34a-1eb45de2a8f9">

**Comment by GeorgeKerscher** [comment]
Marisa, you write that Thorium does not display the footnote properly. Below, James points out his approach and that it works in browsers and in Thorium. Is this the same or different from what you did?

Here is what James emailed me.
Hi George, 

Related to the call yesterday here is the attribute selection CSS to superscript a link tag with the doc-noteref role attribute. This has slight larger text since I do not reduce the size and adds a little margin buffer to make the selection location a little wider, but that could be dropped.

I tested this in Safari, Chrome, and Thorium and they all work. I also noticed your CSS is already using similar attribution selections, though they focus on the epub:type attribute. Either could be used if you wanted to target just one attribute, which would reduce conflicts. 

This is a good method to standardize and automate the formatting, however I expect most vendors will continue to use a superscript tag in the link tag. 

With role: 

a[role="doc-noteref"] {
margin-left:1px;
margin-right:2px;
vertical-align: super;}

With epub:type:
a[epub|type="noteref"] {
margin-left:1px;
margin-right:2px;
vertical-align: super;}


james
